test/powershell/Language/Classes/Scripting.Classes.BasicParsing.Tests.ps1

```diff
@@ -688,6 +688,14 @@ Describe 'Type building' -Tags "CI" {
             ++$a::a | Should Be 2
         }
     }
+
+    It 'should get the script from a class type' {
+        class C {}
+
+        $a = [C].Assembly.GetCustomAttributes($false).Where{
+            $_ -is [System.Management.Automation.DynamicClassImplementationAssemblyAttribute]}
+        $a.ScriptFile | Should BeExactly (& { $MyInvocation.ScriptName })
```

Comment:
Maybe replace `(& { $MyInvocation.ScriptName })` with `$PSCommandPath`?

Reply:
Uh, sure. My excuse - I've never used it before and was too lazy to look for it. :)